app/views/categories/show.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="flex flex-col h-screen justify-between">
+  <div>
+    <div class="navigation py-6 px-3 blue-nav-white-text">
+      <ul class="flex justify-between items-center">
+        <li class="w-10"><%= link_to image_tag('backarrow.png', alt: 'Back'), categories_path, class: 'nav-link' %></li>
+        <h2><strong><%= @category.name %></strong></h2>
+        <li><%= link_to 'Sign Out', destroy_user_session_path, data: { turbo_method: :delete }, class: "nav-link" %></li>
+      </ul>
+    </div>
+    <p style="color: green"><%= notice %></p>
+
+    <%= render @category %>
+    <section class="px-5">
+      <div id="purchases">
+        <% @purchases.each do |purchase| %>
+          <a href="<%= category_purchase_path(@category, purchase) %>" class="m-1">
+          <%= render "purchases/purchase", purchase: purchase %>
+          </a>
+        <% end %>
+      </div>
+    </section>
+  </div>
+
+  <section class="flex flex-col space-y-2 p-5">
+      <a href="<%= new_category_purchase_path(@category) %>" class="">
+        <button data-testid="new-purchase-button" class="drop-shadow-md p-3 w-full green-btn text-white rounded-md hover:bg-green-600 focus:outline-none focus:bg-green-600">
+          New purchase
+        </button>
+      </a>
```

Comment:
- You have done well so far, but according to the previous reviewer, this button should be named `Add a new transaction `and should be styled according to the given design template. Please fix this 😄